experimental/sdg-tracker/src/state/index.ts

```diff
@@ -34,7 +34,6 @@ import rootTopics from "../config/rootTopics.json";
 import sidebarConfig from "../config/sidebar.json";
 import targetText from "../config/targetText.json";
 import {
-  CONTINENTS,
```

Comment:
The declaration can also be removed: https://github.com/datacommonsorg/website/blob/master/experimental/sdg-tracker/src/utils/constants.ts#L58

Reply:
done!